src/utils/amplitude-utils.ts

```diff
@@ -38,6 +38,11 @@ const logAmplitudeEvent = (eventName: string, data?: { [key: string]: EventDataV
 	})
 }
 
-export const logSidevisning = (pathname: string): void => {
+export const loggSidevisning = (pathname: string): void => {
 	logAmplitudeEvent('sidevisning', { path: pathname })
 }
+
+export const loggKlikk = (komponent: string, verdi?: string, status?: string) => {
+	const eventKategori = 'Brukerklikk'
+	logAmplitudeEvent('navigasjon', { eventKategori, komponent, verdi, status })
```

Comment:
Siden vi måler klikk, burde ikke eventet også hete noe tilsvarende, f.eks 'klikk' eller 'bruker_klikk'

Reply:
Ønsker ikke å lage et skille mellom utifra hva som er resultatet av klikket for å gjøre det enklere å finne igjen det man skal filtrere på. Men det kan godt hete klikk isteden for navigasjon, men vil da ha både de som navigerer og ikke som resultat av klikk, skal bruke samme event.